flink/v1.12/flink/src/main/java/com/netease/arctic/flink/table/descriptors/ArcticValidator.java

```diff
@@ -136,27 +138,55 @@ public class ArcticValidator extends ConnectorDescriptorValidator {
           .defaultValue(false)
           .withDescription("If it is true, Arctic source will generate watermark after stock data being read");
 
+  public static final ConfigOption<String> ARCTIC_EMIT_MODE =
+      ConfigOptions.key("arctic.emit.mode")
+          .stringType()
+          .defaultValue(ARCTIC_EMIT_AUTO)
+          .withDescription("file, log, auto. e.g.\n" +
+              "'file' means only writing data into filestore.\n" +
+              "'log' means only writing data into logstore.\n" +
+              "'file,log' means writing data into both filestore and logstore.\n" +
+              "'auto' means writing data into filestore if the logstore of the arctic table is disabled;" +
+              " Also means writing data into both filestore and logstore if the logstore of the arctic table" +
+              " is enabled.\n" +
+              "'auto' is recommended.");
+
+
+  public static final ConfigOption<Duration> AUTO_EMIT_LOGSTORE_WATERMARK_GAP =
+      ConfigOptions.key("arctic.emit.auto-write-to-logstore.watermark-gap")
+          .durationType()
+          .noDefaultValue()
+          .withDescription("Only enabled when 'arctic.emit.mode'='auto', if the watermark of the arctic writers" +
+              " is greater than the current system timestamp subtracts the specific value, writers will also write" +
```

Comment:
Does the specific value mean the value of "arctic.emit.auto-write-to-logstore.watermark-gap"?

Reply:
Yes.